### DISCUSSION TITLE
扫描/影响行数 有问题

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2298] [General]
SQL检测 扫描/影响行数 有问题，实际上只有两行数据修改，但是检测结果却显示影响行数有1111行。影响行数是 goinception 的返回, 但是goinception 官方 给的解决办法是的修改 调用参数 方式：https://github.com/hanchuanchuan/goInception/issues/492
是在sql执行语句前加注释，在 archery 中也是这么操作吗，还是通过其他方式来修改这个参数，直接在注释里添加似乎不生效？
![企业微信截图_16945765383278](https://github.com/hhyo/Archery/assets/32159630/af6898b1-1df1-4091-b718-0c3ec22022d7)
![企业微信截图_16945765839281](https://github.com/hhyo/Archery/assets/32159630/3a3b04ed-6590-49eb-a7fa-19188fa94a75)
![企业微信截图_16946551218788](https://github.com/hhyo/Archery/assets/32159630/a4142340-3192-46ef-a69c-74794c61b4b3)

**Comment by a8159236** [comment]
![image](https://github.com/hhyo/Archery/assets/32159630/4d3a8ab5-6108-48df-8a18-52f37c662dce)
是通过修改这个参数为true吗？等同于real_row_count ？

**Reply by LeoQuote** [reply]
是的，请参考这里的回复，如有需要自己可以修改，也欢迎提交pr https://github.com/hhyo/Archery/issues/643#issuecomment-591986122

**Reply by a8159236** [reply]
好的谢谢
